packages/amplication-client/src/Resource/git/GitActions/ExistingConnectionsMenu.scss

```diff
@@ -79,7 +80,7 @@
     background-color: var(--black5);
     height: 34px;
     display: flex;
-    color: var(--black80);
+    color: var(--white);
```

Comment:
@martinez-hugo please note that we pushed a big ui change to "next" recently 
it looks like you branched out before the change and the code is not up to date - we also changed the variable names so we don't have --black80 or --white any more

Please pull the latest changes from `next` into your branch or create a new branch from `next` and send a new PR

Reply:
Ok nice ! I delete this PR and I create another. Thanks for all